### PR TITLE
Update media

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Video.md
```
remove `tag = ???` and replace it with `video` and `/video` in the practice question so users don't think `tag` is an attribute
Also, change `* movie` to `/vid` to make it a bit less obvious with only one closing tag option.

Remove duplicate codepen from the insight and leave it only in the link section
```

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/html/media/video.md?ref=27ef4fe462687f655f5399ca29271c5305cde576

File: web/html/media/video-formats.md [hunk 1]
```diff
@@ -35,8 +35,6 @@ As you can see, you really only need 2 HTML5 video formats for modern web browse
 
 This CodePen will provide all the necessary code - it's a decent chuck compared to just the `video` element.
 
-[View CodePen](https://codepen.io/enkidevs/pen/xJbYLE)
```
Comment (sourabhv) [inline web/html/media/video-formats.md]: Why did we delete these links?

Comment (sourabhv) [inline web/html/media/video-formats.md]: @nem035 Should we prefer this codepen button over link, coz we have a custom button for it now?

File: web/html/media/video.md [hunk 2]
```diff
@@ -42,7 +42,6 @@ The `<video>` tag was introduced in HTML5 and is used to embed videos on a web p
...
 
-[View CodePen](https://codepen.io/enkidevs/pen/MBYOJb)
```
Comment (sourabhv) [inline web/html/media/video.md]: @nem035 same here

File: web/html/media/video-formats.md (hunk 1 above)
Comment (nem035) [reply web/html/media/video-formats.md]: Yeah, I think that makes sense. Is the md syntax the button expects in the form of a md link?

Comment (sourabhv) [reply web/html/media/video-formats.md]: yea, a hyperlink at root level is converted into a button

Comment (Stefan-Stojanovic) [reply web/html/media/video-formats.md]: The button (as it is now) is always in the middle of the insight and takes the user away from the app in the middle of reading the content, whereas when its a link in the link section it is located at the bottom and users would first finish the insight and then wander away from the app to test what they've learned

Comment (nem035) [reply web/html/media/video-formats.md]: Valid point, I see.

Comment (nem035) [reply web/html/media/video-formats.md]: Update: based on our latest discussions, we should put this button back

Comment (Stefan-Stojanovic) [reply web/html/media/video-formats.md]: I thought I already posted a PR with the opposite changes done in this one(regarding codepens) and mistakenly never posted that one and thought this one was the new one. I am posting the new one now.

**Pull request closed without merging**